File: cpp/bc2540.cpp
#include <iostream>
#include <sstream>
#include <string>
#include <cmath>

using namespace std;

int main()
{
	int nPe, count, nVo, tVo;
	string line, token, res;
	
	while(cin >> nPe)
	{
	    cin.ignore();
	    
		int vec[nPe];
		count = nVo = 0;
	
		getline(cin, line);
		
		istringstream iss (line);
		
		while(iss >> token)
		{
			vec[count] = stoi(token);	
			count++;
		}
		
		for(int i = 0; i < nPe; i++)
		    if(vec[i] == 1) nVo++;
		    
		tVo = (int) ceil((nPe/3.0)*2);
		
		res = nVo >= tVo? "impeachment" : "acusacao arquivada";
		
		cout << res << endl;
	}
}

File: python/bc2540.py
from math import ceil
while True:
    count = 0
    nVo = 0
    try:
        nPe = int(input())

        vec = [int]*nPe

        line = input().split(" ")

        for c in range(0, nPe):
            vec[c] = int(line[c])

        for d in range(0, nPe):
            if vec[d] == 1: nVo += 1

        tVo = int(ceil((nPe/3)*2))

        res = "impeachment" if nVo >= tVo else "acusacao arquivada"
        print(res)

    except EOFError:
        break